pkg/controller/secret/secret_controller.go

```diff
@@ -497,6 +512,15 @@ func (r *ReconcileSecret) Reconcile(request reconcile.Request) (reconcile.Result
 	pagerDutySecretExists := secretInList(secretNamePD, secretList)
 	snitchSecretExists := secretInList(secretNameDMS, secretList)
 
+	cmList := &corev1.ConfigMapList{}
+	err = r.client.List(context.TODO(), cmList, opts...)
+	if err != nil {
+		log.Error(err, "Unable to list configMaps")
+	}
+
+	// Check for the presence of specific configMaps.
+	namespaceCMExists := cmInList(cmNameManagedNamespaces, cmList)
+
 	// Get the secret from the request.  If it's a secret we monitor, flag for reconcile.
 	instance := &corev1.Secret{}
 	err = r.client.Get(context.TODO(), request.NamespacedName, instance)
```

Comment:
Does this work if the type of object that triggered the reconcile is a configmap?

Reply:
Yeah, this function will return true if, any time the reconcile loop is run, there exists a configmap named after the variable `cmNameManagedNamespaces`. 

The `reconcile` loop now runs every time a configmap or secret changes, before it only would run if a secret changes.

Reply:
Ah, I just realized you meant line 526, not 522, my bad. 
The block just continues if it can't find a secret with the request name, so it doesn't affect reconciles with the `managed-namespaces` ConfigMap. I've added a corresponding block to check for ConfigMap errors as well.